dev/ci/docker/bionic_coq/Dockerfile

```
@@ -12,14 +12,14 @@ RUN apt-get update -qq && apt-get install --no-install-recommends -y -qq \
         # Dependencies of lablgtk (for CoqIDE)
         libgtksourceview-3.0-dev \
         # Dependencies of stdlib and sphinx doc
-        texlive-latex-extra texlive-fonts-recommended texlive-xetex latexmk \
+        texlive-latex-extra texlive-fonts-extra texlive-xetex latexmk \
         xindy python3-pip python3-setuptools python3-pexpect python3-bs4 \
         # Dependencies of source-doc and coq-makefile
         texlive-science tipa
 
 # More dependencies of the sphinx doc
-RUN pip3 install sphinx==1.8.0 sphinx_rtd_theme==0.2.5b2 \
-                 antlr4-python3-runtime==4.7.1 sphinxcontrib-bibtex==0.4.0
+RUN pip3 install sphinx==2.3.1 sphinx_rtd_theme==0.4.3 \
+                 antlr4-python3-runtime==4.7.1 sphinxcontrib-bibtex==0.4.2x
```

Still failing because of a silly typo here.

```suggestion
                 antlr4-python3-runtime==4.7.1 sphinxcontrib-bibtex==0.4.2
```